### ISSUE TITLE
Saving the cache to the SD card

### ISSUE THREAD

**Issue by Prikolist2021** [z-huang/music#1584]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Is it possible to add the ability to save the cache to the SD card?

### Why do you want this feature?

Do not take up my phone's memory and save the downloaded files after wipe /data

**Comment by Elizabeth-420xx** [comment]
Yes this is the only feature I really need! Like youtube music does
Downloads take up way too much space on my phone memory,😭

**Comment by androidghosh** [comment]
Needed
